doc/articles/dummies1.md

```diff
@@ -0,0 +1,143 @@
+
+### update Uno.UI/Uno.Core to last dev version
+Or maybe feature is added, and you can use it - with small risk that something is broken.
+Check "next version" section in previously mentioned file: [ReleaseNotes](https://github.com/unoplatform/uno/blob/master/doc/ReleaseNotes/_ReleaseNotes.md).
+If it seems that your problem is resolved, install latest dev version (in Manage NuGet Packages for Solution, check 'include prerelease' checkbox).
+
+### check PR (Pull Requests)
+Your next hope is that someone already make necessary changes to Uno, but these changes are not included even in dev version (yet). It means that some discussion about implementation is under way, or maybe some tests are failing. But you can expect this feature in some near time appear in at least dev version.
+
+List of open PR can be found here: [Pull Requests](https://github.com/unoplatform/uno/pulls).
+
+### consider workaround
+Maybe feature is not implemented, but some "very close" feature is. For example, maybe your app is using CalendarDatePicker (not implemented in Uno 1.45). But Uno implement DatePicker, and this allows to select date, and besides that, this implementation in many ways resemble CalendarDatePicker from UWP.
+So, you can use CalendarDatePicker in UWP, and DatePicker in all other. It can be done using XAML prefixes, in this case:
+
+`<win:CalendarDatePicker Name="yourName" DateChanged="myDateChanged" ... />`
+
+`<non_win:DatePicker Name="youName" DateChanged="myDateChangedDP" ... />`
+
+and add second event handler, myDateChangedDP - it has to be new handler, as CalendarDatePicker.DateChanged and DatePicker.DateChanged handlers have different argument types.
+
+All prefixes list: [platform-specific-xaml](https://github.com/unoplatform/uno/blob/master/doc/articles/platform-specific-xaml.md).
+
+### search for NuGet package for Xamarin
+Maybe your app sends Toasts. They are not implemented in Uno, but you can use this: https://github.com/edsnider/LocalNotificationsPlugin .
+Similar solution exist for using Clipboard, and probably for many more problems.
+
+### create an Issue (feature request)
+If all this fails, you can request adding new feature to Uno.
+
+Before submitting new Issue, check if someone else doesn't submit such issue before: [Issues](https://github.com/unoplatform/uno/issues).
+
+If not, use this: [new feature request](https://github.com/unoplatform/uno/issues/new?labels=kind%2Fenhancement%2C+triage%2Funtriaged&template=enhancement.md).
+
+### contribute to Uno
+This is best (or: preferable) option, if using newer Uno versions / checking existing PR doesn't solve your problem. Best, but also it requires much more time from your side.
+But this is outside scope of this article.
+
+### make strip-down version
+As last resort, you can make some app functions work only under Windows, and WASM/iOS/Android versions somehow limited.
+
+
+Now it is time for compiling and test for platforms other than UWP.
```

Comment:
Everything from this point onward belongs in a separate guide

Reply:
Maybe yes... but I think it is too short for separate article. And it nicely stands as 'next step' from contents above :)

Reply:
There's nothing wrong with short articles :) It's long enough. (And it may get longer if someone else comes along and decides to add more detail. ;) ) You can add a link to it as a 'next step'.

Reply:
Ok, but where? As another article in 'Using Uno'?

I almost can write article about 'contributing to Uno', or maybe: draft, to much rework in process, as I'm not sure my 'procedure' of contributing is good (whats more, I'm almost sure that it has errors :) )

Reply:
Yes it'd be in the 'Using Uno' section.

> I almost can write article about 'contributing to Uno', or maybe: draft, to much rework in process, as I'm not sure my 'procedure' of contributing is good (whats more, I'm almost sure that it has errors :) )

If you don't feel confident to write the article yourself, you could create an issue for what you'd like to see added, along with whatever material you already have. :)

Reply:
So, split article dummies1.md to dummies1.md and dummies.droid.md? :)

Create issue or PR with proposed text of third part?

Reply:
> So, split article dummies1.md to dummies1.md and dummies.droid.md? :)
> 
> Create issue or PR with proposed text of third part?

Yes, but don't use 'dummies' - call them `howto-migrate-existing-app.md` and... `running-android.md`, let's say.

Reply:
Should article for android be in separate PR?

Reply:
Whatever suits you better. If you make a separate PR for it, this PR will be merged that much quicker ;)

Reply:
One question more... before I delete this content.
Isn't it natural after:
"Now it is time for compiling and test for platforms other than UWP."? 
And step-by-step procedure how to compile/test for Android?

Reply:
Yeah, by 'from this point onward' I meant including that line.

Or, yes it's a natural 'next step', but it also belongs in a separate document. But that's the beauty of hyperlinks. :)

Reply:
Then it seems like 'article without end' :)
But Ok.
First this PR, than PR2079, then reset my fork/clone of Uno, and new PR for 'android', android geolocator, storagefile/folder methods...

Check changes (adding 'head' info), maybe some other changes would be needed... every time I read some of my text, I want to change something... but, happily, these texts are already published in journal, and cannot be changed :)